cmd/root.go

```diff
@@ -157,6 +157,12 @@ or ./scorecard --{npm,pypi,rubgems}=<package_name> [--checks=check1,...] [--show
 		}
 		switch format {
 		case formatDefault:
+			score, e := repoResult.AggregatedScore(checkDocs)
```

Comment:
Why are we only calculating this only for default format, we should add it everywhere. Does adding a field in scorecard result hard (like does it need migration), otherwise just add it. we would need some aggregate result everywehre.

Reply:
I can do that, I was mostly hesitant to update the JSON format. Do you want to have this reflected in the BQ table too?

Reply:
ack. That'll come in a follow-up PR.